README-Milestone1.md

```diff
@@ -0,0 +1,133 @@
+# Milestone 1 - Create a Single Node Uni
+In this section, we will deploy a new Uni called "test-heros-react"
+
+## Explore the /uni_configuration directory
+The /uni_configuration directory contains several pre-created files that will help us quickly create a new Uni 
+
+* __schema.json__ - Contains the data model for this Uni, which defines two types of characters: Heros :) and Villians :(.
+* __initial-state.json__ - Contains seed data for this Uni, which creates several characters for us to use in workshop
+* __registration.singlenode.json. - Contains cloud, region, and account information for this Uni
+
+## Create the Uni
+Now it's time to create a Uni based on the files from the previous section.
+
+If not already logged in to the share service do so by running [`share login`](https://vendia.net/docs/share/cli/commands/login)
+
+```bash
+share login
+```
+  
+The `share uni create` command creates the Uni.  For this step, we'll use the `registration.singlenode.json` file.  Pick a unique `name` for your Uni  that begins with `test-`.  By default, all Unis share a common namespace so here is your chance to get creative.  Update the `userId` attribute of the node to reflect your Vendia Share username (i.e. your email address).
```

Comment:
I haven't tried using email addresses in uni names.  Adjust this one to use the username portion of their email to be sure.

Reply:
I've separated the Uni name updating sentence from the userId updating sentence.  I intended those to be two separate updates but realize now that wasn't clear since they were appended.

Reply:
Ah - gotcha.  Makes sense.